tests/bud.bats

```diff
@@ -4254,6 +4254,9 @@ _EOF
   run_buildah 125 build $WITH_POLICY_JSON -t ${target} --pull=false $BUDFILES/pull
   expect_output --substring "busybox: image not known"
 
+  run_buildah build $WITH_POLICY_JSON -t ${target} --pull=missing $BUDFILES/pull
+  expect_output --substring "COMMIT pull"
+
   run_buildah build $WITH_POLICY_JSON -t ${target} --pull $BUDFILES/pull
```

Comment:
This was previously going to _have_ to pull the image in order for the build to complete at all, but the newly-inserted `--pull=missing` run will make that unnecessary, so it either needs to specify a different base image, or it needs to be removed in between.